paper-menu-button.html

```diff
@@ -305,6 +305,17 @@
         },
 
         /**
+         * Toggles the drowpdown content between opened and closed.
+         */
+        toggle: function() {
```

Comment:
Is this new method actually necessary to correct the test? I ask because the default behavior of an open "overlay" in our domain is to close automatically if it detects a click "outside" of it.